Makefile

```diff
@@ -35,16 +35,19 @@ pentest:
 qa:
 	$(eval env=qa)
 	$(eval env_config=qa)
+	$(eval backend_key=-backend-config=key=${env}-bat.terraform.tfstate)
```

Comment:
For permanent environments, the key is hard coded in the *_backend.tfvars.
Ex for qa: https://github.com/DFE-Digital/register-trainee-teachers/blob/master/terraform/workspace-variables/qa_backend.tfvars#L4
So this looks redundant?